src/__test__/calculate.test.js

```diff
@@ -0,0 +1,76 @@
+import calculate from '../logic/calculate';
+
```

Comment:
- [ ] Please add tests for all the functionalities of your calculate script, this includes +/-, division, and division by 0 operations to comply with the project requirements: "All the functionalities & scenarios of the operate.js and calculate.js modules should be covered using Jest."

Reply:
We added the required test to the calculate script, thank you.